backend/src/main/java/WebApi.java

```diff
@@ -7,9 +10,51 @@
     private static WebInterface webInterface = new WebInterfaceImpl();
 
     public static void main(String[] args) {
+        WebApi.enableCORS("*","*","*");
+
+        Gson gson = new Gson();
         get("/hello", (req, res) -> "Hello World");
 
-        get("/stl/:stlID", (req, res) -> webInterface.getAllPrintsForStl(req.params("stlID")));
+        // Customers
+        get("/customers", (request, response) -> webInterface.getAllCustomers(), gson::toJson);
+        get("/customers/:customerID,", ((request, response) -> webInterface.getCustomer(request.params(":customerID"))), gson::toJson);
+        get("/customers/:customerID/orders,", ((request, response) -> webInterface.getOrdersFromCustomer(request.params(":customerID"))), gson::toJson);
+        get("/customers/:customerID/digitalparts,", ((request, response) -> webInterface.getDigitalPartsFromCustomer(request.params(":customerID"))), gson::toJson);
+        get("/customers/:customerID/physicalparts,", ((request, response) -> webInterface.getPhysicalPartsFromCustomer(request.params(":customerID"))), gson::toJson);
+        post("/customers", ((request, response) -> webInterface.createNewCustomer(gson.fromJson(request.body(), Customer.class))), gson::toJson);
+        put("/customers", ((request, response) -> webInterface.updateCustomer(gson.fromJson(request.body(), Customer.class))), gson::toJson);
+        delete("/customers/:customerID,", ((request, response) -> webInterface.deleteCustomer(request.params(":customerID"))), gson::toJson);
+
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)